packages/dmn-js-decision-table/src/features/powered-by/components/PoweredByLogoComponent.js

```diff
@@ -20,7 +20,7 @@ export default class PoweredByLogoComponent extends Component {
     return (
       <div className="powered-by">
         <div
-          className="powered-by__logo"
+          className="logo"

```

Comment:
If possible I'd love to keep this aligned and/or align it with `bpmn-js` and the DRD viewer:

```html
<a href="http://bpmn.io" target="_blank" class="bjs-powered-by" title="Powered by bpmn.io" style="color: rgb(64, 64, 64); position: absolute; bottom: 15px; right: 15px; z-index: 100;"><svg>...</svg></a>
```

Reply:
OK let's use `style` everywhere. I think it makes sense.

Reply:
:+1: